src/babashka/curl.clj

```diff
@@ -127,15 +127,22 @@
 (defn- read-headers [^File header-file]
   (line-seq (io/reader header-file)))
 
+(defn- read-then-unread
+  [^java.io.InputStream is]
+  (let [c    (.read is)
+        bais (when-not (= -1 c)
```

Comment:
Maybe it'd be nice if the boolean `(= -1 c)` was named and then re-use that name in the steps after it.

Reply:
Good call out.